pkg/services/serviceaccounts/database/database.go

```diff
@@ -247,6 +247,35 @@ func (s *ServiceAccountsStoreImpl) UpdateServiceAccount(ctx context.Context,
 	}, err
 }
 
+func (s *ServiceAccountsStoreImpl) SearchOrgServiceAccounts(ctx context.Context, query *models.SearchOrgUsersQuery) ([]*serviceaccounts.ServiceAccountDTO, error) {
+	if !query.IsServiceAccount {
```

Comment:
I would instead set:

`query.IsServiceAccount = true` 

to ensure IsServiceAccount is always set here instead of returning an error

Reply:
hmm, but would not that mean someone that calls this without it will get service accounts instead.